File: c/4-functions/practice/fp-4.c
#include <stdio.h>
int main()
{
     int p(int a)
    {
            int n =a*(3*(a) -1);
            int p =n/2;
            return p;
    }
    int a;
    scanf("%d",&a);
    if(a<0)
    {
        printf("Invalid Input");
    }
    else
    {   for(int i=1;i<=a;i++)
        {
        int result =p(i);
        printf("%d\n",result);
        }
    }
}

File: python/5-functions/practice/fp-4.py
import math 
def cubes(m,n):
    if m>n or m<0 or n<0 :
        return "Invalid input"
        exit()
    else:
        sum=0
        for i in range(m,n+1):
            cubes =pow(i,3)
            sum+=cubes
        return sum
m=int(input())
n=int(input())
result=cubes(m,n)
print(result)